src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/MakeAnApplicationService.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.*;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.field.IdValue;
+
+@Service
+public class MakeAnApplicationService {
+    public String getMakeAnApplicationTypeName(AsylumCase asylumCase) {
+
+        Optional<List<IdValue<MakeAnApplication>>> maybeMakeAnApplications = asylumCase.read(AsylumCaseDefinition.MAKE_AN_APPLICATIONS);
+
+        if (maybeMakeAnApplications.isPresent()) {
+            List<IdValue<MakeAnApplication>> makeAnApplications = maybeMakeAnApplications.orElse(Collections.emptyList());
+            if (makeAnApplications.size() > 0) {
+                return makeAnApplications.get(0).getValue().getType();
```

Comment:
is it always first in the row, what happens when the decision is refused?

Reply:
yes, the current application is always added the beginning of the list:
see this file for reference
https://github.com/hmcts/ia-case-api/pull/568/files#diff-650d0010f8d40cbed667257b0a9503fa70dd0ccfd08c52b72994e8777a2e97f6